src/main/scala/com/gilcloud/sbt/gitlab/GitlabPlugin.scala

```diff
@@ -101,9 +107,8 @@ object GitlabPlugin extends AutoPlugin {
         .value,
       publish := publish.dependsOn(headerAuthHandler).value,
       publishTo := (ThisProject / publishTo).value.orElse {
-        gitlabProjectId.value map { p =>
-          "gitlab-maven" at s"https://${gitlabDomain.value}/api/v4/projects/$p/packages/maven"
-        }
+        gitlabProjectId.value.map(p => "gitlab-maven" at s"https://${gitlabDomain.value}/api/v4/projects/$p/packages/maven") orElse
+        gitlabGroupId.value.map(g => "gitlab-maven" at s"https://${gitlabDomain.value}/api/v4/groups/$g/-/packages/maven")
```

Comment:
are you able to publish to group endpoint?

Reply:
AFAIK it is gitlab premium feature